tools/customization_dataset_preparation/customization_dataset_preparation.py

```diff
@@ -367,7 +398,7 @@ def print_all_messages(messages):
     messages = []
     messages.append(str(args))
 
-    MAX_TOTAL_CHAR_LENGTH = 10000
+    MODEL_MAX_SEQ_LENGTH = 2048
```

Comment:
The Rope models can support 4096 tokens (and even beyond that)

Reply:
Will update when they get released!